launch/cape_ros_node.launch

```diff
@@ -0,0 +1,10 @@
+<launch>
+
```

Comment:
To include this launch file and pass ns as its parameter, we need to first declare argument so line 4 has to be up here.

Reply:
this is addressed on the branch versions/1.0